src/test/java/org/andersen/service/impl/HotelServiceImplTests.java

```diff
@@ -0,0 +1,53 @@
+package org.andersen.service.impl;
+
+import org.andersen.model.Apartment;
+import org.andersen.model.Hotel;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class HotelServiceImplTests {
+
+    @Mock
+    private Hotel hotel;
+
+    @InjectMocks
+    private HotelServiceImpl hotelService;
+
+    private Apartment apartment;
+
+    @BeforeEach
+    void setup() {
+        apartment = new Apartment(50);
+        when(hotel.getApartments()).thenReturn(new ArrayList<>());
```

Comment:
same

Reply:
should i move that to test methods? or how can i return an array for tests?

Reply:
leave it please)